pkg/server/container_create_windows.go

```diff
@@ -402,7 +400,8 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 			userstr = image.Config.User
 		}
 		if userstr != "" {
-			g.AddAnnotation("io.microsoft.lcow.userstr", userstr)
```

Comment:
I hope no one is actually setting this annotation manually.

Reply:
First thing I thought of too, I'd hope not. Be weird when they could use the run_as_username cri field

Reply:
Could you double check through our logs?

Reply:
No one should be setting this after following up